_posts/2022-04-08-mfa-phase-1-rollout.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Making popular Ruby packages more secure
+layout: post
+author: 
+author_email: 
```

Comment:
Not sure who should be the author. Thinking it should be someone from the RubyGems Team?

Reply:
You can be the author. If you prefer, you can add your team name as well.